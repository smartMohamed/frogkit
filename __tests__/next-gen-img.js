import { mount } from 'vue-test-utils'
import nextGenImg from '@/components/next-gen-img'

describe('Logistic Badges component', () => {

  it('should render component and match snapshot', () => {
    const wrapper = mount(nextGenImg, {propsData: {src: "http://images.ctfassets.net/6iactef8ko6c/51BTxQjQM84MaMGEk4cY4k/18956cef0802562b46a72ee665dadabb/sf_cam1_402x.jpg", alt: 'smartfrog cam'}})
    expect(wrapper.html()).toMatchSnapshot()
  })

})
