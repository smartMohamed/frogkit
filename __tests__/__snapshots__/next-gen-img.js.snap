// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logistic Badges component should render component and match snapshot 1`] = `
<picture>
    <source srcset="http://images.ctfassets.net/6iactef8ko6c/51BTxQjQM84MaMGEk4cY4k/18956cef0802562b46a72ee665dadabb/sf_cam1_402x.jpg?fm=webp" type="image/webp">
    <source srcset="http://images.ctfassets.net/6iactef8ko6c/51BTxQjQM84MaMGEk4cY4k/18956cef0802562b46a72ee665dadabb/sf_cam1_402x.jpg" type="image/jpeg"> <img src="http://images.ctfassets.net/6iactef8ko6c/51BTxQjQM84MaMGEk4cY4k/18956cef0802562b46a72ee665dadabb/sf_cam1_402x.jpg" alt="smartfrog cam"></picture>
`;
